projects/ngrx.io/content/guide/router-store/install.md

```diff
@@ -0,0 +1,17 @@
+# Installation
```

Comment:
We also have a ng-add command for router-store.

Reply:
Same here, tried running but getting command not found.

Reply:
Here's the message I get...
```sh
Collection "@ngrx/router-store" cannot be resolved.
Error: Collection "@ngrx/router-store" cannot be resolved.
    at NodeModulesEngineHost._resolveCollectionPath (C:\users\wgrimes\sandbox\ngrx-test-ng-add\node_modules\@angular-devkit\schematics\tools\node-module-engine-host.js:91:15)
    at NodeModulesEngineHost.createCollectionDescription (C:\users\wgrimes\sandbox\ngrx-test-ng-add\node_modules\@angular-devkit\schematics\tools\file-system-engine-host-base.js:111:27)
    at SchematicEngine._createCollectionDescription (C:\users\wgrimes\sandbox\ngrx-test-ng-add\node_modules\@angular-devkit\schematics\src\engine\engine.js:147:40)
    at SchematicEngine.createCollection (C:\users\wgrimes\sandbox\ngrx-test-ng-add\node_modules\@angular-devkit\schematics\src\engine\engine.js:140:43)
    at AddCommand.getCollection (C:\users\wgrimes\sandbox\ngrx-test-ng-add\node_modules\@angular\cli\models\schematic-command.js:123:35)
    at AddCommand.runSchematic (C:\users\wgrimes\sandbox\ngrx-test-ng-add\node_modules\@angular\cli\models\schematic-command.js:262:50)
    at AddCommand.run (C:\users\wgrimes\sandbox\ngrx-test-ng-add\node_modules\@angular\cli\commands\add-impl.js:48:31)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:744:11)
    at startup (internal/bootstrap/node.js:285:19)
```

Reply:
That's odd. Its showing correctly when I run `npm show @ngrx/router-store`

Reply:
@brandonroberts @timdeschryver I'm still having an issue getting these to work but I will go ahead and update the docs and push a new commit to this PR so it can get merged.

I'll open a separate issue for the `ng add` errors so that this doesn't get mixed up anymore.

Reply:
Sounds good @wesleygrimes !

Reply:
No need to create an issue - https://github.com/ngrx/platform/pull/1526